mis_builder/static/src/xml/mis_widget.xml

```diff
@@ -42,7 +42,7 @@
                                     </a>
                                 </t>
                                 <t t-if="!value_value.drilldown">
-                                    <t t-esc="value_value.val_r"/>
+                                    <t t-esc="value_value.val_r" t-field-options='{"widget": "monetary"}'/>
```

Comment:
Sorry. I missed this part in my review before fast-tracking, but I think it doesn't hurt, and most as they're the authors of the module.